android/build.gradle

```diff
@@ -13,6 +13,9 @@ buildscript {
 apply plugin: 'com.android.library'
 apply from: '../common.gradle'
 
+sourceCompatibility = 1.7
```

Comment:
Given that we're testing now against 1.8, couldn't we use that as well for source code? We may want to use 1.8 features.

Reply:
It may not be that simple. It would be great to use 8 but there will likely be implications in respect of our Android support (we want to go back to API Level 19).